cfme/automate/service_dialogs.py

```diff
@@ -155,6 +158,24 @@ def update(self, updates):
         sel.click(form_buttons.save)
         flash.assert_no_errors()
 
+    def update_element(self, second_element, element_data):
+        sel.force_navigate('service_dialog_edit', context={'dialog': self})
+        if version.current_version() > "5.5":
+            tree = accordion.tree("Dialog")
+        else:
+            tree = Tree("dialog_edit_treebox")
+        tree.click_path(self.label, self.tab_label, self.box_label)
+        self.add_element(second_element)
+        list_ele = []
+        list_ele.append(element_data.get("ele_label"))
+        list_ele.append(second_element.get("ele_label"))
+        tree.click_path(self.label, self.tab_label, self.box_label)
+        ele_1 = self.element(list_ele[0])
+        ele_2 = self.element(list_ele[1])
+        sel.drag_and_drop(ele_1, ele_2)
+        sel.click(form_buttons.save)
+        flash.assert_no_errors()
```

Comment:
@mfalesni @RonnyPfannschmidt (FW3.0) perhaps this assertion could be part of a standard decorator for UI methods that always runs a wait_for_ajax and a flash assert after performing a UI operation?